multimodal-search-tirg/README.MD

```diff
@@ -64,7 +65,8 @@ If it's running successfully, you should be able to see logs scrolling in the co
 </p>
 
 ## Query
-This will start the server, run your query and give result as a pop-up.
+This will start the server, run your query and give result as a pop-up. TIRG Multimodal encoder requires both image and text to create final encoding. This is possible by leveraging our MultiModalEncoder capabilities. We QueryLanguageDriver to redirect text and image documents based on modality.  
```

Comment:
We QueryLanguageDriver isn't it a little weird to usa an infinitive as a verb here?

Reply:
LoL... It was supposed to be We `use` QueryLanguageDriver